src/app/components/MediaLoader/configs/legacyMedia.ts

```diff
@@ -0,0 +1,45 @@
+import filterForBlockType from '#lib/utilities/blockHandlers';
+import {
+  LegacyMediaBlock,
+  ConfigBuilderProps,
+  ConfigBuilderReturnProps,
+} from '../types';
+import buildPlaceholderConfig from '../utils/buildPlaceholderConfig';
+
+export default ({
+  blocks,
+  basePlayerConfig,
+  translations,
+}: ConfigBuilderProps): ConfigBuilderReturnProps => {
+  const legacyMediaBlock: LegacyMediaBlock =
+    filterForBlockType(blocks, 'legacyMedia') ?? {};
+
+  const { image, format, playlist } = legacyMediaBlock?.content ?? {};
+
+  const placeholderConfig = buildPlaceholderConfig({
+    title: '',
+    type: format || 'video',
+    holdingImageURL: image?.href,
+    translations,
+  });
+
+  const mp4Items = playlist
+    ?.filter(item => item.format === 'mp4')
+    ?.map(item => ({ href: item.url }));
+
+  return {
+    mediaType: format || 'video',
+    playerConfig: {
+      ...basePlayerConfig,
+      playlistObject: {
+        title: '',
```

Comment:
I think this should have a value rather than be '': https://github.com/bbc/morph-modules/blob/0f16bd34fad7339f0a61e5dcff3e489dbba9de3d/news-av-player/src/legacy-media/index.js#L24

Reply:
Good shout, I'll pass it in as an override from the main article headline